pkg/template/helper.go

```diff
@@ -123,6 +123,7 @@ func currentProductVersion() (string, error) {
 	if err != nil {
 		return "", shared.ReturnLogError("failed to get product: %w", err)
 	}
+	fmt.Println(version)
```

Comment:
why this print?

Reply:
To display the version on the logs, helps adding it to issue validation

Reply:
why have that already under debug flag here 
https://github.com/rancher/distros-test-framework/blob/b098805b7b4c04d8d638593adc23eb8e71e1d04a/pkg/template/processor.go#L86

 we also have it here - https://github.com/rancher/distros-test-framework/blob/b098805b7b4c04d8d638593adc23eb8e71e1d04a/entrypoint/versionbump/versionbump_suite_test.go#L81


but if you want add it yet , lets try to add as log not printing , since doesnt need any especial formatting , lets try to keep consistency using logs to return info, not print

Reply:
Go it, I was not seeing it getting printed after this command was executed on the logs 
```
18:16:56  PASSED! Flannel Version bump: Validate Pod
18:16:56  [38;5;10m•[0m[37mDEBU[0m[2024-06-14T23:16:42Z] Execute: rke2 -v on <ip-addr>           
18:16:56  
18:16:56  Command:
```
so I thought may be it was not implemented. Will update

Reply:
sorry got confused lol

do you still need to print? if so , lets use log.level()

Reply:
Yes I think we should print, updated